### PR TITLE
MC-2287 Rebranding form fields styles

### PR DESCRIPTION
**Pull request by emrberk**

### CODE REVIEW

**Review by utkukaratas — CHANGES_REQUESTED** [review]
you'll need to regenerate the reference screenshots in the repo. it's a bitch :/

File: packages/ui/styles/constants/_colors.scss [hunk 1]
```diff
@@ -26,3 +26,12 @@ $colorAccessibilityOutline: #9648bd !default;
 $colorBrand: #00bac6 !default;
 $colorShadow: rgb(53 61 73 / 20%) !default;
 $colorMenuDark: #333 !default;
+$colorDanger: #dc084b !default;
+$colorGrayDarkest: #323b40 !default;
+$colorGrayDark: #576770 !default;
+$colorGrayLight: #c2cbd5 !default;
+$colorGrayLightest: #f7faff !default;
+$colorCustom5: #0019fe !default;
```
[the remark this answers is not in the document]
Comment (emrberk) [reply packages/ui/styles/constants/_colors.scss]: Removed new colors except custom6. The new colors need to be approved before implemented.

File: packages/ui/styles/constants/_colors.scss [hunk 2]
```diff
@@ -26,3 +26,4 @@ $colorAccessibilityOutline: #9648bd !default;
 $colorBrand: #00bac6 !default;
 $colorShadow: rgb(53 61 73 / 20%) !default;
 $colorMenuDark: #333 !default;
+$colorCustom6: #f0f6ff !default;
```
Comment (fantkolja) [inline packages/ui/styles/constants/_colors.scss:29]: Would be great to come-up with a more meaningful name

File: packages/ui/styles/constants/_sizes.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 $grid: 0.25rem !default;
 $alertWidth: 350px !default;
-$borderRadius: $grid * 1.25 !default;
+$borderRadius: $grid * 2 !default;
```
Comment (fantkolja) [inline packages/ui/styles/constants/_sizes.scss:8]: It will change a lot of other border-radiuses. Is @pawelklasa OK with that?

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: packages/ui/styles/constants/_sizes.scss (hunk 3 above)
Comment (emrberk) [reply packages/ui/styles/constants/_sizes.scss:8]: It is adapted from the design (border-radius: 8px). Let's wait for a response from @pawelklasa

Comment (fantkolja) [reply packages/ui/styles/constants/_sizes.scss:8]: Most probably he won't answer here, it's better to ask him in the https://hazelcast.slack.com/archives/C01BT11F24E

File: packages/ui/styles/constants/_colors.scss (hunk 2 above)
Comment (fantkolja) [reply packages/ui/styles/constants/_colors.scss:29]: As discussed [here](https://hazelcast.slack.com/archives/C01BT11F24E/p1691670433506009?thread_ts=1691668858.529289&cid=C01BT11F24E) I would redefine `$colorOverlayBackground`

Comment (Sodik) [reply packages/ui/styles/constants/_colors.scss:29]: `$colorOverlayBackground` - but we use it not (only) for overlay. Let it be `$colorPrimaryBackground`

Comment (fantkolja) [reply packages/ui/styles/constants/_colors.scss:29]: I mean, that we have already [`$colorOverlayBackground`](https://github.com/hazelcast/hive/blob/master/packages/ui/styles/constants/_colors.scss#L24). Let's just redefine its value to `#f0f6ff` and use it as form background.

File: packages/ui/styles/constants/_sizes.scss (hunk 3 above)
Comment (pawelklasa) [reply packages/ui/styles/constants/_sizes.scss:8]: Those look good to me.  @emrberk @fantkolja

File: packages/ui/styles/constants/_colors.scss (hunk 2 above)
Comment (emrberk) [reply packages/ui/styles/constants/_colors.scss]: Merged `$colorOverlayBackground` with `$colorCustom6`.

**Review by fantkolja — APPROVED** [review]

**Review by utkukaratas — APPROVED** [review]

**Pull request merged**